### PR TITLE
Elec 127

### PR DESCRIPTION
**Pull request by timmy-huang**
Made code pretty uwu

### CODE REVIEW

File: src/components/dropdown-sort-menu.js [hunk 1]
```diff
@@ -1,31 +1,10 @@
...
-  },
-  {
-    key: 'Lowest Difficulty',
-    text: 'Lowest Difficulty',
-    value: 'Lowest Difficulty',
-  },
-];
-
-const DropdownSort = () => {
+const DropdownSort = (props) => {
```
Comment (hayeselnut) [inline src/components/dropdown-sort-menu.js:6]: rename component or file to match

File: src/pages/home-page.js [hunk 2]
```diff
@@ -13,62 +13,46 @@ import DropdownCourseSearchSelection from '../components/searchbar.js';
...
+  'Lowest Difficulty',
 ];
 
+const majorOptions = majors.map((item) => createDropdownOption(item));
+
+const termOptions = terms.map((item) => createDropdownOption(item));
+
+const sortOptions = sorts.map((item) => createDropdownOption(item));
+
+console.log(majorOptions);
```
Comment (hayeselnut) [inline src/pages/home-page.js:54]: remove console.log

**Review by hayeselnut — CHANGES_REQUESTED** [review]

File: src/pages/home-page.js [hunk 3]
```diff
@@ -5,70 +5,52 @@ import { Dropdown, Header, Input } from 'semantic-ui-react';
...
-    key: 'Programming Languages',
-    text: 'Programming Languages',
-    value: 'Programming Languages',
-  },
-  {
-    key: 'Security Engineering',
-    text: 'Security Engineering',
-    value: 'Security Engineering',
-  },
+const createDropdownOption = (Item) => {
```
Comment (hayeselnut) [inline src/pages/home-page.js:16]: Lowercase

**Review by hayeselnut — CHANGES_REQUESTED** [review]

**Comment by timmy-huang** [comment]
I think i have fixed everything but it still says changes requested im not sure what to do

**Review by hayeselnut — CHANGES_REQUESTED** [review]
lint

**Review by hayeselnut — APPROVED** [review]
finally

**Pull request merged**